### ISSUE TITLE
Remove excess lefthand padding/margin in ordered lists in Formation

### ISSUE THREAD

**Issue by GnatalieH** [department-of-veterans-affairs/vets-design-system-documentation#1362]
## Description
The excess lefthand padding/margin space in ordered lists in Formation creates a situation where about 25% of the lefthand side of the screen is taken up by white space in the mobile view. This leads to more scrolling and is bad for readability on mobile. We should remove unnecessary padding, which is also closer to how USWDS does it.

## Details
Current state in Production:

![localhost_3001_decision-reviews_supplemental-claim_file-supplemental-claim-form-20-0995_form-saved (1).png](https://images.zenhubusercontent.com/6111980ab6b167c27c9a5723/87e9405b-5ba0-4c3f-8740-c95b1239a7aa)

Suggested improvement:

![Screen Shot 2022-11-28 at 2.37.34 PM.png](https://images.zenhubusercontent.com/6111980ab6b167c27c9a5723/d853263f-73ca-498a-bb59-8f79a68355a5)

USWDS example:

![Screen Shot 2022-11-28 at 2.35.02 PM.png](https://images.zenhubusercontent.com/6111980ab6b167c27c9a5723/9cd54716-3982-4b84-a678-fc0bd7af7966)

##Tasks
1. Research how making this fix in Formation will impact things site-wide
2. Remove excess margin/padding for ordered lists in Formation

**Comment by caw310** [comment]
@LillyBoot @babsdenney , do we still need this or can we close?

**Comment by babsdenney** [comment]
@caw310 @LillyBoot  Looking at the documentation this is still an issue. We can keep this open.  

[VADS Typography](https://design.va.gov/foundation/typography#lists)

**Comment by caw310** [comment]
@babsdenney do we still need issue?

**Comment by babsdenney** [comment]
Yes still needed, I discussed this with @danbrady and @LWWright7 and we can expand this ticket to:
- Remove all additional styles added to [VADS lists](https://design.va.gov/components/list) and align with the [USWDS list styles](https://designsystem.digital.gov/components/list/). So no more square bullets or extra spacing on the list. 
- Remove the "List" navigational menu on the [Typography](https://design.va.gov/foundation/typography) page.
